rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -44,7 +44,7 @@ class SE3 {
         var ones = nj.ones([1, 1]);
         var homogeneous_pt = nj.concatenate(point, ones);
 
-        var point_world_coords = homogeneous_pt.dot(this.transform_matrix.T);
```

Comment:
this was right before I believe -- why the change?
```
p_w = wTc * p_c
```
(4,1) = (4,4) * (4,1)
or
p_w = (wTc * p_c)^T
p_w^T = p_c^T (wTc)^T
(1,4) = (1,4) * (4,4)
See Boyd's Crimes against Matrices: http://ee263.stanford.edu/notes/matrix_crimes.pdf

So we can do
(N,4) = (N,4) * (4,4)

However, we actually dont need the homogenous thing at all, we can just do 
```python
return point_cloud @ self.rotation.T + self.translation
```

Do we have a unit test on this?

Reply:
Thanks for breaking this down and sending the link. Not sure why I changed it, but I've reverted the change. I do not have unit tests for `se3.js`, so I will add those in.